test/src/sender/main.cpp

```diff
@@ -78,63 +95,83 @@ using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
-string input_data_control = "test/data/sender/sender_input_test_control_In.txt";
-const char * i_input_data_control = input_data_control.c_str();
+    string input_data_control =
+    	"test/data/sender/sender_input_test_control_In.txt";
+    const char * i_input_data_control = input_data_control.c_str();
 
-std::shared_ptr<cadmium::dynamic::modeling::model> generator_con = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator_con" , std::move(i_input_data_control));
+    std::shared_ptr<cadmium::dynamic::modeling::model> generator_con =
+        cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen,
+	    TIME, const char* >("generator_con" ,std::move(i_input_data_control));
 
-string input_data_ack = "test/data/sender/sender_input_test_ack_In.txt";
-const char * i_input_data_ack = input_data_ack.c_str();
+    string input_data_ack = "test/data/sender/sender_input_test_ack_In.txt";
```

Comment:
all the input and output paths are hard coded here, we prefer soft coding . Using #define to represent the text would be a better way!!
Please check for other files as well

Reply:
changed after review

Reply:
verified